src/types/FormTypes.ts

```diff
@@ -10,14 +10,23 @@ export interface SignUpFormProps {
   password: string;
   confirmPassword: string;
 }
+
+export interface PasswordFormValues {
+  password: string;
+  confirmPassword: string;
+}
+
+export interface EmailFormValues {
+  email: string;
+}
 export interface RecipeFormValues {
-  title: string,
-  time: string,
-  description: string,
-  image?: FileList,
-  preparing: Preparing[],
+  title: string;
+  time: string;
+  description: string;
+  image?: FileList;
+  preparing: Preparing[];
   ingredients: Ingredient[];
-  imgSrc: string
+  imgSrc: string;
```

Comment:
code formatter didn't catch that earlier? 🤔

Reply:
I think so??? For sure I didn't change it manually

Reply:
That's what I hoped for 😅